sql/2020/20_Caching/content_age_older_than_ttl.sql

```diff
@@ -0,0 +1,31 @@
+#standardSQL
+# Requests with a content age older than its TTL
+CREATE TEMPORARY FUNCTION toTimestamp(date_string STRING)
+RETURNS INT64 LANGUAGE js AS '''
+  try {
+    var timestamp = Math.round(new Date(date_string).getTime() / 1000);
+    return isNaN(timestamp) ? -1 : timestamp;
+  } catch (e) {
+    return -1;
+  }
+''';
+
+SELECT
+  client,
+  COUNT(0) AS total_req,
+  COUNTIF(diff < 0) AS req_too_short_cache,
+  ROUND(COUNTIF(diff < 0) * 100 / COUNT(0), 2) AS perc_req_too_short_cache

```

Comment:
(throughout) leave the numbers unformatted in SQL and format them in the results sheet
```suggestion
  COUNTIF(diff < 0) / COUNT(0) AS perc_req_too_short_cache

```

Reply:
Removed across the queries. Percentage calculation being done in google sheet.